Britannia/events/BTWK_frontier_events.txt

```diff
@@ -569,31 +574,41 @@ BTWK_frontier_event.0204 = {
 						triggered_desc = {
 							trigger = {
 								scope:restored_hadrians_wall_major_character = {
-									NOT = {
+									NOR = {
 										has_title = title:e_britannia
+										has_title = title:e_roman_britannia
 									}
 								}
-								NOT = {
-									capital_province = {
-										geographical_region = world_caledonia
+								OR = {  # Can't receive this desc if not imperial-like culture in Caledonia
```

Comment:
Think this comment should be `# Will only receive this desc if in Caledonia and not imperial-like culture ` ?

Reply:
` # Receive this desc given you are NOT a "Pictish" (non-Imperial-like) Caledonian` is probably more apt.